runtime/android/core_internal/src/org/xwalk/core/internal/XWalkContent.java

```diff
@@ -57,7 +59,7 @@
     private static Class<? extends Annotation> javascriptInterfaceClass = null;
 
     private ContentViewCore mContentViewCore;
-    private ContentView mContentView;
+    private XWalkContentView mContentView;
```

Comment:
We need to override `ContentView.onCreateInputConnection` to redirect it to `XWalkView`. The route is `XWalkContentView` -> `XWalkView` -> (If the user doesn't override it) -> `XWalkContent` -> `ContentView`.